tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -157,3 +187,25 @@ def set_packet_props_full_relativity(r_packet, numba_model):
     r_packet.nu *= inverse_doppler_factor
     r_packet.energy *= inverse_doppler_factor
     r_packet.mu = (r_packet.mu + beta) / (1 + beta * r_packet.mu)
+
+
+def packet_tracking(r_packet):
```

Comment:
Is this the best location for this function?

Reply:
Moved to another file 😄